pkgs/development/python-modules/simple-term-menu/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pyproject = true;
+  nativeBuildInputs = [ setuptools ];
+
+  disabled = pythonOlder "3.4";
+
+  pname = "simple-term-menu";
+  version = "1.6.4";
+
+  src = fetchPypi {
```

Comment:
```
  pname = ...;
  version = ...;
  pyproject = ...;
  disabled = ...;

  src = ...;

  nativeBuildInputs = ...;
  ...
```
In practice, the order does not affect the packaging itself, but it is preferable to do it like other packages to make code review and maintenance easier.

Reply:
Makes sense, I have updated the order.